README.md

```diff
@@ -59,7 +59,27 @@ pipenv run python manage.py runserver
 and see a local version of the website following `http://127.0.0.1:8000/` on
 the browser.
 
-After made code changes and before commit, check code style.
+Open another terminal and navigate to webpack folder using:
+```shell
+cd webpack
+```
+
+To install webpack dependencies use:
+```shell
+npm install
```

Comment:
Tracking the `package-lock.json` file will allow to do `npm ci` instead, installing the project dependencies based on this file.

Reply:
Yeah. I have updated the readme